d3a_api_client/utils.py

```diff
@@ -166,6 +167,21 @@ def get_area_uuid_and_name_mapping_from_simulation_id(collab_id, domain_name):
         return area_name_uuid_map
 
 
+def get_aggregators_list(domain_name=None):
+    """
+    Returns a list of aggregators for the logged in user
+    """
+    if not domain_name:
+        domain_name = os.environ.get("API_CLIENT_DOMAIN_NAME")
+    query = 'query { aggregatorsList { configUuid name  devicesList { deviceUuid } } }'
+
+    data = post_graphql_request(domain_name=domain_name, query=query)
+    if key_in_dict_and_not_none(data, "errors"):
+        return ast.literal_eval(data["errors"][0]["message"])
+    else:
+        return data["data"]["aggregatorsList"]
```

Comment:
```suggestion
    return ast.literal_eval(data["errors"][0]["message"]) if key_in_dict_and_not_none(data, "errors") else return data["data"]["aggregatorsList"]
```
Could be one liner here.